packages/clerk-js/src/ui/hooks/useFetch.ts

```diff
@@ -17,7 +17,7 @@ export const useFetch = <T>(
 
   const fetcherRef = useRef(fetcher);
 
-  useEffect(() => {
+  useLayoutEffect(() => {
```

Comment:
@panteliselef Are we confident that this change won't have any negative impact on performance?

Reply:
reverting back to useEffect